beacon/endpoints/rest/__init__.py

```diff
@@ -57,18 +57,27 @@ class GVariantParametersBase(RequestParameters):
     includeDatasetResponses = ChoiceField("NONE", "ALL", "HIT", "MISS", default="NONE")
     assemblyId = RegexField(r'^((GRCh|hg)[0-9]+([.]?p[0-9]+)?)$', ignore_case=True, default=None)
     variantType = ChoiceField("DEL", "INS", "DUP", "INV", "CNV", "SNP", "MNP", "DUP:TANDEM", "DEL:ME", "INS:ME", "BND")
-    filters = ListField(items=RegexField(r'.*:.+=?>?<?[0-9]*$'), default=None)
+
+    # Examples:
+    # PATO:0000011<=P70Y
+    # HP:0100526-
+    # HP:0005978
+    # HP:0012622&sim=low
+    # HP:0012622&sim=medium
+    # HP:0012622&sim=high
+    # HP:0032443="unknown medical history"
+    # HP:0032443=%"unknown medical history"%
+    # HP:0032443=!"unknown medical history"
+    # filters = ListField(items=RegexField(r'^.*:\w+(>|<)?=?P?[0-9]+Y?$'), default=None)
+    filters = ListField(items=RegexField(r'^.*:\w+(((>|<)?=?(P?[0-9]+Y?))|-|&sim=(low|medium|high)|=(%|!)?"[0-9a-zA-Z\s]+"%?)?$'), default=None)
```

Comment:
Are you covering that complex regular expression with some tests? If not, you should write some in the same PR.

Reply:
Moreover, I think it's better to implement a similar logic for filter than we did for the list of request fields. You should have a `NumericFilter`, `OntologyFilter`, etc...

Reply:
After some thoughts, implementing the above amounts into reinventing the regex. But more limited.
Not sure it's worth it.